.github/workflows/code_review.yaml

```
@@ -0,0 +1,70 @@
+name: Code Review
+
+concurrency:
+  group: ci-${{ github.head_ref }}-${{ github.base_ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+
+  compile:
+    name: Compile project
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@c85c95e3d7251135ab7dc9ce3241c5835cc595a9 # v3.5.3
+
+      - name: Setup Node.JS
+        uses: ./.github/actions/setup-node
+
+      - name: Check lock file
+        run: npx -y package-lock-utd@1.1.3
+
+      - name: Check for new dependencies
+        uses: hiwelo/new-dependencies-action@e4175e14484328f9ab56df91e420e90ad5031c08 # v1.0.1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install dependencies
+        run: npm ci --audit=false --fund=false
+
+      - name: Compile
+        run: npm run compile
+
+      - name: Store in cache
+        uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8 # v3.3.1
+        id: restore-build
+        with:
+          path: ./*
+          key: ${{ github.sha }}
+
+  code_checks:
+    name: Code checks
+    runs-on: ubuntu-latest
+
+    needs: compile
+
+    steps:
+      - name: Restore cache
+        uses: actions/cache@88522ab9f39a2ea568f7027eddc7d8d8bc9d59c8 # v3.3.1
+        id: restore-build
+        with:
+          path: ./*
+          key: ${{ github.sha }}
+
+      - name: Check lint rules
+        run: npm run lint
+
+# Uncomment this step when there will be tests
+#      - name: Run tests
+#        run: npm run test
```

It is already working, we can already run the command! It produce the following output:

```
• Packages in scope: 
• Running test in 0 packages
• Remote caching disabled

No tasks were executed as part of this run.

 Tasks:    0 successful, 0 total
Cached:    0 cached, 0 total
  Time:    21ms 
```

```suggestion
      - name: Run tests
        run: npm run test
```